htdocs/core/lib/signature.lib.php

```diff
@@ -79,7 +79,7 @@ function getOnlineSignatureUrl($mode, $type, $ref = '', $localorexternal = 1)
 	if ($type == 'proposal') {
 		$securekeyseed = isset($conf->global->PROPOSAL_ONLINE_SIGNATURE_SECURITY_TOKEN) ? $conf->global->PROPOSAL_ONLINE_SIGNATURE_SECURITY_TOKEN : '';
 
-		$out = $urltouse.'/public/onlinesign/newonlinesign.php?source=proposal&ref='.($mode ? '<span style="color: #666666">' : '');
+		$out = $urltouse.'/public/onlinesign/newonlinesign.php?mode=proposal&ref='.($mode ? '<span style="color: #666666">' : '');
```

Comment:
Parameter for "newonlinsesign.php" is "source"
parameter for ajax "onlineSign.php" is mode.
So old code seems good.